maro/simulator/scenarios/data_center/topologies/azure.2019.original/config.yml

```diff
@@ -1,8 +1,8 @@
+# Initial buffer time budget.
+buffer_time_budget: 0
 # The number of ticks to delay when agent decides not to assign VM.
 delay_duration: 1
-# The calibration parameter used for the CPU utilization vs energy consumption model.
-calibration_parameter: 1.4
-# The idle power usage of a machine.
-busy_power: 10
-# The busy power usage of a machine.
-idle_power: 1
+
+vm_table: "/mnt/d/kuanwei/data/test_336k/build/vmtable.bin"
```

Comment:
- refer to citi_bike, relative path like ``~/.maro/...`` would be better
- add annotation for these file path
- For you to run pass, if relative paths don't work for you, locally modify them to the absolute path